internal/command/jsonplan/plan.go

```diff
@@ -574,6 +581,18 @@ func MarshalResourceChanges(resources []*plans.ResourceInstanceChangeSrc, schema
 	return ret, nil
 }
 
+func marksContainEphemeral(addr addrs.AbsResourceInstance, valMarks []cty.PathValueMarks) error {
```

Comment:
ensureEphemeralMarksAreValid?

Something that makes it sound more like a check

Reply:
Done in 50de180e54a5645c3184c57a2e8bd2ff1badfbba